chart/values.yaml

```diff
@@ -0,0 +1,141 @@
+# Default values for tocloud9.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+redis:
+  enabled: true
+  architecture: standalone
+  auth:
+    enabled: false
+nats:
+  enabled: true
+mysql:
+  enabled: true
+  global:
+    storageClass: my-storage-class
+  initdbScriptsConfigMap: mysql-init-scripts
+
+databases:
```

Comment:
It would be nice to provide other params like `hostname` and `port` in case if database is outside of the cluster.